cloud-sql/sqlserver/mssql/Dockerfile

```diff
@@ -0,0 +1,26 @@
+# Copyright 2022 Google LLC. All rights reserved.
+# Use of this source code is governed by the Apache 2.0
+# license that can be found in the LICENSE file.

```

Comment:
Can you make this match our normal license headers?

Reply:
Updated.